internal/jstest/jstest.go

```diff
@@ -34,6 +35,12 @@ func Run(ctx context.Context, dir string, args []string) (*internal.TestResults,
 
 	filesM := make(map[string]struct{})
 
+	cwd, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+
+	os.Chdir("../")
```

Comment:
I think, in principle, `os.Chdir` can return an error. Do we ignore it on purpose?

Also, I see we use `../` as path, I expect it might not work on Windows, are we ok with it? 

Also, I'm not sure we this command is called here. Why do we need to call it?

Reply:
We change to the parent directory so we can match files using globs and have the filenames be correct when passed to the container. The filenames also need to match so we don't fail when we check the diff. 

Maybe I should put this back into draft so I can find a simpler solution? cc. @AlekSi.